src/handlers/payout/action.ts

```diff
@@ -74,9 +74,8 @@ export const handleIssueClosed = async () => {
   logger.info(`Posting a payout url to the issue, url: ${payoutUrl}`);
   const comment = `### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
   const comments = await getAllIssueComments(issue.number);
-  const commentContents = comments.map((i) => i.body);
-  const exist = commentContents.find((content) => content.includes(comment));
-  if (exist) {
+  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
```

Comment:
@rndquu this seems too brittle. 

We should actually parse the URL with the URL constructor, and then work backwards from there.

Reply:
Yes, parsing the URL with `URLSearchParams` is a better approach

This PR solves the original issue + we don't have plans of adding more query params besides `claim` and `network` so I don't think that refactoring to `URLSearchParams` is an urgent issue

> Besides, isn't it supposed to be `https://pay.ubq.fi/?claim=`

No, the generated URL is `https://pay.ubq.fi?claim=` so everything is fine